client/js/components/toggle-show-hide.js

```diff
@@ -0,0 +1,22 @@
+import showHide from './show-hide';
+import { trackEvent } from '../utils/track-event';
+
+export default (el) => {
+  const toggle = showHide({el});
+  const trackingJson = el.getAttribute('data-tracking-info');
```

Comment:
Sorry to be pedantic, but why not store the track info here?
Just wondering if we could just use 1 paradigm (either generic tracking, or specific)?